boards/xtensa/intel_adsp_cavs25_tgph/intel_adsp_cavs25_tgph.dts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2020,2022 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <intel/intel_cavs25_tgph.dtsi>
+
+/ {
+	model = "intel_adsp_cavs25_tgph";
+	compatible = "intel";
```

Comment:
I don't see much point to this compatible, but it's not worth blocking over.

Reply:
I kept this still as the same compatible is in all cavs variants. better to remove from all if we remove.